### ISSUE TITLE
PackageInstall jobs do not work in MiniShift

### ISSUE THREAD

**Issue by displague** [crossplane/crossplane#1402]
### What happened?

While attempting to debug https://github.com/crossplane/stack-aws-sample/issues/17, @muvaf discovered that the StackInstall InstallJobs had abbreviated pods logs. This created incomplete CRD documents and lead to the omission of other CRD resources and the Stack resources itself.

The `stack unpack` command (the install job) works by having the package mounted to it, examining that package, and outputting the appropriate Kubernetes resources to the pod log.

A truncation or rotation of this log will result in a failure to install a Stack.

#### How can we reproduce it?

* Install Minishift
* Install crossplane
* Attempt to StackInstall `provider-aws`

#### What environment did it happen in?

Minishift v1.34.2
Crossplane version: v0.9.0

### How can Crossplane help?

Can container logs sizes be defined by the Stack Manager when creating install jobs?  Can these ultimately be overruled by policies?

Can the Stack Manager use an alternative approach to recording the resources of a Stack? Perhaps a ConfigMap?

**Comment by muvaf** [comment]
@displague does this happen only in Minishift or is OpenShift also affected?

**Comment by displague** [comment]
This issue is tracking what we noticed in minishift. Open shift is not affected from the limited environments I have experienced.

I don't know about other environments. Presumably the setting that minishift uses will prevent stackinstall from working under similar settings. 

This issue is a place to explore the reason (identify the setting) and possible solutions.

OpenShift does not appear to be affected, so I am not personally affected by this limitation.
